ci-operator/step-registry/baremetalds/e2e/upgrade/baremetalds-e2e-upgrade-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: baremetalds-e2e-upgrade
+  steps:
+    allow_skip_on_success: true 
+    pre:
+    - ref: baremetalds-packet-setup
+    - ref: ipi-install-rbac
+    - ref: baremetalds-devscripts-setup
+    test:
+      - ref: baremetalds-e2e-conf
+      - ref: baremetalds-e2e-test
+      - ref: baremetalds-e2e-upgrade-upgrade
```

Comment:
See ci-operator/step-registry/openshift/upgrade/aws/openshift-upgrade-aws-workflow.yaml - reuse `openshift-e2e-test` with parameters

Reply:
Are you maybe referring to the usage of `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE`?

Reply:
This var would be automatically set by ci-operator